### PR TITLE
Add emailEventHistory method to org, vapp and vdc

### PR DESCRIPTION
**Pull request by JoMas971**

### CODE REVIEW

File: src/sdk/model/org/org.ts [hunk 1]
```diff
@@ -1800,4 +1800,16 @@ export class Org extends Entity {
     });
   }
 
+  /**
+   * Retrieve a CSV report email with all event history for a given Org.
+   * @returns {Promise<Task>} task promise
+   */
+  /* istanbul ignore next: autogenerated */
+  async emailEventHistory(email: string): Promise<Task> {
```
Comment (cfsnyder) [inline src/sdk/model/org/org.ts]: did you autogenerate these @JoMas971 ?

Comment (JoMas971) [reply src/sdk/model/org/org.ts]: no indeed... I've just copy paste it... Will need to write the tests then.

Comment (cfsnyder) [reply src/sdk/model/org/org.ts]: i ask because the signature is wrong, these endpoints don't return tasks

Comment (JoMas971) [reply src/sdk/model/org/org.ts]: yeah... I just saw this... The vm signature was already there, so i just copy pasted it on other entity but i didn't see that the signature was wrong. I'm fixing it and add some tests ;-).

Comment (cfsnyder) [reply src/sdk/model/org/org.ts]: why not just actually generate them using the plugin?

**Pull request merged**